spec/controllers/tasks_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+describe TasksController do
+  let(:user) { create(:user) }
+  let(:task) { create(:task) }
+
+  describe '#index' do
+    let(:task) { create(:task, title: 'test') }
+
+    context 'when the user is logged in' do
+      include_context :logged_in_user
+
+      context 'when search params are provided' do
```

Comment:
I'm confused why you're testing for search params when there's no search functionality on this page so far.